SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -240,31 +255,39 @@ Interest InterestFactory::CreateServerWorkerInterest()
 	return ServerInterest;
 }
 
-Interest InterestFactory::CreateInterest() const
+Interest InterestFactory::CreateInterest(Worker_EntityId EntityId) const
```

Comment:
Nit: another const

Reply:
just made all the worker entity IDs const all over the place